config/scribe.php

```diff
@@ -176,7 +176,6 @@
     'try_it_out' => [
         /**
          * Add a Try It Out button to your endpoints so consumers can test endpoints right from their browser.
-         * Don't forget to enable CORS headers for your endpoints.
          */
```

Comment:
We shouldn't remove this. The user still needs to have CORS enabled server-side, or else this will not work. Laravel Sanctum is not the only API client.